cpp/src/DataStorm/CallbackExecutor.cpp

```diff
@@ -31,18 +31,18 @@ CallbackExecutor::CallbackExecutor(function<void(function<void()> call)> callbac
                 }
 
                 lock.unlock();
-                for (const auto& p : queue)
+                for (const auto& callback : queue)
                 {
                     try
                     {
                         if (_callbackExecutor)
                         {
-                            // TODO do we need to ensure p.second is executed before we continue?
-                            _callbackExecutor(p.second);
+                            // TODO do we need to ensure that the callback is executed before we continue?
+                            _callbackExecutor(callback);
```

Comment:
I would prefer a clearer name for this field.

Reply:
Renamed to `_customExecutor`